openedx/core/djangoapps/user_authn/tasks.py

```diff
@@ -69,7 +69,7 @@ def send_activation_email(self, msg_string, from_address=None):
     except AttributeError:
         site = Site.objects.get_current()
 
-    user = User.objects.get(username=msg.recipient.username)
+    user = User.objects.get(username=msg.recipient.lms_user_id)
```

Comment:
Please, change it to the original definition: https://github.com/Pearson-Advance/edx-platform/blob/pearson-release/olive.main/openedx/core/djangoapps/user_authn/tasks.py#L66 @sergivalero20

Reply:
@Squirrel18 understood!